services/frontend-service/pkg/handler/handleEnvironments.go

```diff
@@ -35,6 +35,8 @@ func (s Server) HandleEnvironmentGroups(w http.ResponseWriter, req *http.Request
 	switch function {
 	case "locks":
 		s.handleEnvironmentGroupLocks(w, req, envGroup, tail)
+	case "rollout-status":
+		s.handleEnvironmentGroupRolloutStatus(w, req, envGroup)
```

Comment:
please create a simple script like `https://github.com/freiheit-com/kuberpult/blob/main/infrastructure/scripts/create-testdata/create-app-lock.sh` for the new endpoint, so it's easier to run manually